js/src/ankerSnapshot.ts

```diff
@@ -95,7 +105,7 @@ const schema = new Map([
       kind: 'struct',
       fields: [
         ['swapped_rewards_st_sol_total', 'u64'],
```

Comment:
This should be `st_lamports`, right?

Reply:
I've now used the type StLamports for this. Should it still be renamed?